app/controllers/audits_controller.rb

```diff
@@ -4,8 +4,16 @@ class AuditsController < ReactController
   before_action :setup_search_options, :only => :index
 
   def index
-    @audits = resource_base_search_and_page.preload(:user)
-    @host = resource_finder(Host.authorized(:view_hosts), params[:host_id]) if params[:host_id]
+    respond_to do |format|
+      format.html
```

Comment:
i assume this controller shouldn't return any html? in fact it should only handle the search and json payloads for the audit page?

Reply:
Can we use the REST-API to return the data?
I know Graphql does not yet return any audit data, but that should be pretty easy to add as well.

Reply:
AFAIK - there is a lot of logic in rails helpers that require effort to move to the API.. maybe graphql would be easier..?

Reply:
removed `format.html`
now inherits from Application Controller